lib/base-controller.js

```diff
@@ -1,9 +1,24 @@
 'use strict';
 
-var Controller = require('hmpo-form-wizard').Controller;
+var Controller = require('hmpo-date-controller');
 var util = require('util');
 var _ = require('lodash');
 
+// In HOF we are more lenient with date inputs than in vanilla
+// hmpo-form-wizard and so must override some validators
+var validators = Controller.validators;
+validators = _.extend(validators, {
+  'date-year': function hofdateyear(value) {
```

Comment:
Can do. I mimicked the hmpo names I think.

Ideally I'd like to get these versions of the validators into hmpo-form-controller itself as I believe ours are more correct. But they may be being used in ways that prevent this.